docs/getting-started/intro.md

```diff
@@ -15,6 +15,6 @@ Everything you need to know about getting started with Rancher Turtles.
-| Cluster API Operator     | `>=v0.5.1`                               | Using [helm chart](https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/docs/README.md#method-2-use-helm-charts) install or as a [dependency](./install_turtles_operator.md#install-cluster-api-operator-as-a-helm-dependency) (recommended) to the `Rancher Turtles`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Cluster API Operator     | `>=v0.9.1`                               | Using [helm chart](https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/docs/README.md#method-2-use-helm-charts) install or as a [dependency](./install_turtles_operator.md#install-cluster-api-operator-as-a-helm-dependency) (recommended) to the `Rancher Turtles`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | Cluster API              | `v1.4.6`                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Does CAPI version needs to be updated to v1.6.2 ?
Ref: https://github.com/rancher/turtles/pull/406/files

Reply:
@cpinjani no, this has to stay as v1.4.6 which is the version used in our e2e tests and in Rancher itself. The file you are pointing to will be updated to reflect this exact version

Reply:
PR: https://github.com/rancher/turtles/pull/452